app/developers/commands.py

```diff
@@ -114,22 +119,31 @@ def export_grants(grant_ids: list[uuid.UUID]) -> None:  # noqa: C901
                 for form in section.forms:
                     grant_export["forms"].append(to_dict(form))
 
-                    for question in form.questions:
-                        grant_export["questions"].append(to_dict(question))
+                    def add_all_components_flat(
```

Comment:
Does this need to be declared inline? It can just be a function declared once at the module level and take all the bits in it needs?